.clang-format

```diff
@@ -1,2 +1,5 @@
 BasedOnStyle: chromium
+IndentWidth: 8
+UseTab: Always
+BreakBeforeBraces: Linux
```

Comment:
Could `BreakBeforeBraces: Linux` be removed please, the others can stay if you'd like.

Reply:
OK. I also added:
+IndentCaseLabels: false
+PointerAlignment: Right
+AllowShortIfStatementsOnASingleLine: WithoutElse
+PenaltyBreakBeforeFirstCallParameter: 100
+PenaltyBreakAssignment: 100
+PenaltyExcessCharacter: 10

And fixed all my changes to conform to this style.
However, I excluded src/inotifywait.c and src/inotifywatch.c from style check 
because they are space indented and I didn't want to change that.

Reply:
@ericcurtin are you fine with me leaving just tab indentation as your first comment? Because in a later comment you requested to remove all edits from this file.